File: converted_source/updform.pas.java
// unit Updform

from conversion_common import *;
import uhints;
import delphi_compatability;


class PdForm extends TForm {
    
    public void create(TComponent anOwner) {
        super.create(anOwner);
        uhints.makeAllFormComponentsHaveHints(this);
    }
    
}

File: converted_source/updform.pas.py
# unit Updform

from conversion_common import *
import uhints
import delphi_compatability

class PdForm(TForm):
    def __init__(self):
        pass
    
    def create(self, anOwner):
        TForm.create(self, anOwner)
        uhints.makeAllFormComponentsHaveHints(self)
        return self
    
